src/display/llemu.cpp

```diff
@@ -50,5 +50,14 @@ void register_btn2_cb(lcd_btn_cb_fn_t cb) {
 std::uint8_t read_buttons(void) {
 	return lcd_read_buttons();
 }
+
+void set_background_color(lv_color_t color) {
+	lcd_set_background_color(color);
+}
+
+void set_background_color(uint8_t r, uint8_t g, uint8_t b) {
+	lcd_set_background_color(lv_color_t{r,g,b,0xff});
```

Comment:
```suggestion
	lcd_set_background_color(LV_COLOR_MAKE(r, g, b));
```

Should prob have this use the macro as well, just so the config for alpha works.